src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
```suggestion
```

Reply:
what should i cancel here?

Reply:
bc why you need it here?

Reply:
the empty line should be removed. please check other files as well